front/pages/api/apps/[user]/[sId]/runs/index.js

```diff
@@ -49,124 +51,190 @@ export default async function handler(req, res) {
     case "POST":
       if (readOnly) {
         res.status(401).end();
-        break;
+        return;
       }
+      return postHandler(req, res, app, user);
```

Comment:
Out of consistency in code let's unwrap these functions. Not saying we shouldn't do it, but I think it should be done consistently everywhere.

Reply:
Did that to avoid nesting switch statements (as you requested a switch on `mode`). I'll change it back to if/else if/else

Reply:
is it find if I keep `executeModeHandler` and `designModeHandler` ?
I find the function is quite hard to read otherwise